sigmastate/src/main/scala/sigmastate/serialization/SelectFieldSerializer.scala

```diff
@@ -1,18 +1,22 @@
 package sigmastate.serialization
 
 import sigmastate.Operations.SelectFieldInfo
-import sigmastate.Values.Value
+import sigmastate.Values.{Value, SValue}
 import sigmastate.lang.Terms._
 import sigmastate.utils.{SigmaByteReader, SigmaByteWriter}
 import sigmastate.utxo.SelectField
 import sigmastate.{STuple, SType}
+import SelectFieldInfo._
+import sigmastate.utils.SigmaByteWriter.DataInfo
 
 case class SelectFieldSerializer(cons: (Value[STuple], Byte) => Value[SType]) extends ValueSerializer[SelectField] {
   override def opDesc = SelectField
+  val inputInfo: DataInfo[SValue] = inputArg
```

Comment:
@greenhat Yeah, maybe, when reviewing at github. In IDEA it is not so much an issue, because "jump to definition" works pretty fine. Not sure it is worth adding an issue even.

Reply:
It’s not that big of a deal, but even in IDEA it requires an additional action to understand the code. IMO in this case the drawbacks outweighs the benefits.